molecule/default/requirements.yml

```
@@ -1,11 +1,9 @@
 ---
-- src: https://github.com/cisagov/ansible-role-backports
-  name: backports
-- src: https://github.com/cisagov/ansible-role-pip
-  name: pip
-- src: https://github.com/cisagov/ansible-role-python
-  name: python
-- src: https://github.com/cisagov/ansible-role-remove-python2
-  name: remove_python2
-- src: https://github.com/cisagov/ansible-role-upgrade
-  name: upgrade
+- name: backports
+  src: https://github.com/cisagov/ansible-role-backports
+- name: python
+  src: https://github.com/cisagov/ansible-role-pip
+- name: pip
+  src: https://github.com/cisagov/ansible-role-python
```

You done got these mixed up:
```suggestion
- name: pip
  src: https://github.com/cisagov/ansible-role-pip
- name: python
  src: https://github.com/cisagov/ansible-role-python
```